packages/roosterjs-editor-dom/lib/utils/adaptFontColorToBackgroundColor.ts

```diff
@@ -0,0 +1,87 @@
+const enum ColorTones {
+    BRIGHT,
+    DARK,
+}
```

Comment:
If there are only two possible values, why not just use boolean?
We should only use enum when there are at least 3 values.

Reply:
The color can be bright or dark or neither of them, so we have three options

Reply:
But the enum ColorTones only has two values: BRIGHT and DARK.